src/main/java/racing/view/ResultView.java

```diff
@@ -1,24 +1,37 @@
 package racing.view;
 
+import racing.domain.RacingCars;
 import racing.domain.RacingCar;
 
-import java.util.List;
-
 public class ResultView {
     private static final String HYPHEN = "-";
+    private static final String COMMA = ",";
```

Comment:
```suggestion
    private static final String COMMA = "|";
```

`comma`가 본질적인 의미일까요? 
값이 변경되면 이름도 변경해주어야 할까요!?